### DISCUSSION TITLE
Call to undefined method LivewireManager::styles() after upgrading to Laravel 11

### DISCUSSION THREAD

**Discussion by ipkpjersi** [livewire/livewire#8668] [1. Help]
Hi,

After upgrading to Laravel 11 on my project on this branch shift-124802: https://gitlab.com/ipkpjersi/Website-Portal/-/merge_requests/3 I am now getting an error: `Call to undefined method Livewire\LivewireManager::styles() `

![image](https://github.com/user-attachments/assets/fd81d624-928d-4b43-a2c7-343a9919ec47)

Is this an expected error? Did I do something wrong?

I'm using Laravel Mix v6.0.49 if that matters.

**Comment by ipkpjersi** [comment]
I removed the livewireStyles and livewireScripts Blade directives but then I realized oh yeah, my Livewire script isn't working most likely because I believe I was on Livewire 2 and now Livewire is at version 3. 

Instead of going through a second upgrade migration in the middle of another upgrade migration, I ended up rewriting this code without Livewire for now since it was simple enough and the only place that Livewire was currently being used in this codebase. I'm certain I'll use Livewire again in the future but I'll have to learn more about the latest version first some time.

I'll go ahead and close this.

**Comment by seongbae** [comment]
I just started getting this as well after upgrading to Laravel 11 and Livewire 3.  I simply removed the directives because documentation shows that livewire css and javascript are automatically loaded.
